doc/rst/language/spec/modules.rst

```diff
@@ -292,17 +309,106 @@ it contains are accessible via the use statement (:ref:`Using_Modules`), import
 statement (:ref:`Importing_Modules`), or qualified
 naming (:ref:`Explicit_Naming`).
 
+.. _Using_And_Importing:
+
+Using and Importing
++++++++++++++++++++
+
+The ``use`` statement and the ``import`` statement are the two primary ways to
+access a module's symbols from outside of the module.  For top-level modules, a
+``use`` or ``import`` statement is required before referring to the module’s
+name or the symbols it contains within a given lexical scope.
+
+The names that are made visible by a ``use`` or ``import`` statement are
+inserted in to a new scope that immediately encloses the scope within which the
+statement appears.  This implies that the position of the ``use`` or ``import``
+statement within a scope has no effect on its behavior.  If a scope includes
+multiple ``use`` statements, multiple ``import`` statements, or a combination of
+``import`` and ``use`` statements, then the newly-visible names are inserted
+into a common enclosing scope.
+
+.. _Use_And_Import_Conflicts:
+
+Conflicts
++++++++++
+
+The scope inserted by the presence of ``use`` and ``import`` statements follows
+the same rules about conflicting symbols as symbols defined in the same scope
+(see :ref:`Variable_Conflicts`).  Thus an error will be signaled if multiple
+conflicting public module-level symbols would be inserted into this enclosing
```

Comment:
Why not say "Thus an error will be signaled if multiple variables with the same name would be inserted into this implicit scope and that name is accessed".

Reply:
lol, good question XD I'll do that